db/migrate/20171102155207_create_projects.rb

```diff
@@ -4,8 +4,8 @@ def change
       t.text :project_title
       t.string :donors, null: false
       t.string :status, null: false
-      t.date :start_date, null: false
-      t.date :end_date, null: false
+      t.string :start_date, null: false
+      t.string :end_date, null: false
```

Comment:
Can you please change the dates to be integer instead?
Integer would be better for comparison, sorting etc.